mst/security.md

```diff
@@ -146,3 +146,8 @@ account when securing your data. To learn more about security options within Tim
 visit the [Managed TimescaleDB Knowledge Base][timescale-mst-kb].
 
 [cloud-security-eval]: https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwjtm4bbn4rbAhUBDZoKHdBRDgkQFggpMAA&url=https%3A%2F%2Fwww.elfgroup.fi%2Fecc%2F1708-S6-71acd0046.pdf&usg=AOvVaw2wcBEPGeys6PL21W3G6wGW
+
+[timescale-mst-portal]: https://portal.managed.timescale.com/
+[cidr-wiki]: https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing
+[cidr-tool]: FIXME
```

Comment:
@alexthinath I believe this page came from the MST Knowledge Base, though I couldn't find the exact article. Do you remember which CIDR tool we recommended? The link isn't in this file.

Reply:
Maybe I'm just bad at github which article is this refactoring?

This is the only one I can find that mentions CIDR blocks:
https://kb-managed.timescale.com/en/articles/5452806-attaching-a-vpc-to-aws-transit-gateway

Reply:
Same! [This page](https://docs.timescale.com/mst/latest/security/) was created as part of a block of pages that were moved from the KB, so I thought it must've been on the KB at some point. But I can't find it! 

Do you know who I can ask about the CIDR tool? (tagged you because you were author on the articles, but happy to rope someone else in too 😉 )

Reply:
The specific section that discusses CIDR is [here](https://docs.timescale.com/mst/latest/security/#advanced-managed-service-for-timescaledb-configuration). It's about configuring MST network policy with a list of allowed IP addresses.

Reply:
@alexthinath just bumping this, if you can't remember what on earth the tool is supposed to be or who would know, I can just remove the mention of it entirely

Reply:
yeah I'm fine to remove it